resources/js/features/achievements/components/ReportIssueMainRoot/SessionDrivenIssueListItems.tsx

```diff
@@ -0,0 +1,72 @@
+import type { FC } from 'react';
+
+import { buildTrackingClassNames } from '@/common/utils/buildTrackingClassNames';
+import { TicketType } from '@/common/utils/generatedAppConstants';
+import { usePageProps } from '@/features/user/settings/hooks/usePageProps';
+
+import { buildStructuredMessage } from './buildStructuredMessage';
+import { ReportIssueOptionItem } from './ReportIssueOptionItem';
+
+export const SessionDrivenIssueListItems: FC = () => {
+  const { achievement, hasSession, ticketType, extra } =
+    usePageProps<App.Platform.Data.ReportAchievementIssuePageProps>();
+
+  // Don't allow the player to create tickets if they've never loaded the game.
+  if (!hasSession) {
+    return null;
+  }
+
+  if (ticketType === TicketType.DidNotTrigger) {
```

Comment:
This doesn't seem to be working.
![image](https://github.com/user-attachments/assets/6d9e2196-2212-4d32-b6c6-9d4907157dff)

An achievement I haven't unlocked should have did not trigger and triggered but reset items.

Reply:
Looks like it was an issue in the controller. Resolved in https://github.com/RetroAchievements/RAWeb/pull/2676/commits/ffc2aa21d03d67d5d38120220f98e0742033f2bb.